doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md

```diff
@@ -11,7 +11,10 @@ sort_idx: 10
 * TOC
 {:toc}
 
-This guide describes how to use *C SDK* on linux-based systems.
+The application code based on Kaa C SDK is similar for most of the platform used.
+However, build system is not.
```

Comment:
probably should be moved to the main C SDK page with the corresponding links list to the platform-dependent build procedure descriptions